aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/activities/CustomTabsRedirectActivity.java

```diff
@@ -0,0 +1,26 @@
+package com.amazonaws.mobile.client.activities;
+
+import android.app.Activity;
+import android.os.Bundle;
+
+import com.amazonaws.mobile.client.AWSMobileClient;
+import com.amazonaws.mobileconnectors.cognitoauth.activities.CustomTabsManagerActivity;
+
+/**
+ * Handles auth redirect for sign-in and sign-out.
+ */
+public final class CustomTabsRedirectActivity extends Activity {
```

Comment:
Yep, the customer won't be using this file anywhere besides in the manifest. I think it would be better to keep the same name to avoid confusion if anything.